libafl/src/inputs/bytes.rs

```diff
@@ -47,9 +46,7 @@ impl Input for BytesInput {
 
     /// Generate a name for this input
     fn generate_name(&self, _idx: usize) -> String {
-        let mut hasher = AHasher::new_with_keys(0, 0);
-        hasher.write(self.bytes());
-        format!("{:016x}", hasher.finish())
+        format!("{}", Uuid::new_v4().to_simple())
```

Comment:
Isn't it better if the same testcase always has the same name?

Reply:
Idk, it's up to you/ya'll really. The only compelling reason I can think of where I'd want the name to be deterministic is if I were doing testcase lookup based on the filename. I've only ever needed crashing input => filename, but idk if that's something other folks need. 

Do you want to revisit input bytes + `something` to try and get a deterministic hash? 

> ... the .generate_name method is called in `OnDiskCorpus.add`. That method has `self` and a `TestCase` available ...

`TestCase` has an `executions` member, presumably that's a unique-per-core value. What about something like `CORE_ID-HASH-EXECUTIONS`, i.e. `0-56ee067d0349278e-20147`? You'd have the hash value based on testcase input, along with 2 values that should make the filename unique, at least on a per-run basis.